jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_warning/battery_warning.py

```diff
@@ -66,6 +70,20 @@ def speak(self, sentence):
         self.speak_client.send_goal(SoundRequestGoal(sound_request=req))
         self.speak_client.wait_for_result(timeout=rospy.Duration(10))
 
+    def play_sound(self, sound):
```

Comment:
`play_sound` function is only used in a single place. I prefer not to create the function and write the whole code directly where it is used.

Reply:
You are right, so I've made the correction.